lib/internal/worker/io.js

```diff
@@ -85,9 +85,10 @@ const messageTypes = {
   LOAD_SCRIPT: 'loadScript',
 };
 
-let messageEvent;
-function lazyMessageEvent() {
-  return messageEvent ??= require('internal/deps/undici/undici').MessageEvent;
+let fastCreateMessageEvent;
```

Comment:
can you add a comment to here to why it is fast?

Reply:
we skip webidl validations when the arguments are already validated, https://github.com/nodejs/undici/commit/381f32c50e2826760fd399be40e06352f1b8f36d

Reply:
actually the ci isn't green I'll just add a comment in the code, which is probably what you were looking for anyways lol